cursive-core/src/views/select_view.rs

```diff
@@ -957,21 +985,22 @@ impl<T: 'static + Send + Sync> View for SelectView<T> {
                 PaletteStyle::Primary
             };
 
-            let x = match printer.size.x.checked_sub(1) {
-                Some(x) => x,
+            let lw = match printer.size.x.checked_sub(self.decorators_width()) {
```

Comment:
Let's call it `available`, since that's the available width we have for the label.